Makefile

```diff
@@ -19,13 +19,16 @@ define print_header
 	@printf ":: %s\n" ${1}
 endef
 
-CLOE_ROOT   := .
+CLOE_ROOT   := $(shell pwd)
 CLOE_LAUNCH := PYTHONPATH="${CLOE_ROOT}/cli" python3 -m cloe_launch
 
 # Build configuration:
-BUILD_DIR     := build
-INSTALL_DIR   := /usr/local
-CONAN_OPTIONS :=
+BUILD_DIR       := build
+LOCKFILE_SOURCE := conanfile.py
+BUILD_LOCKFILE  := ${BUILD_DIR}/conan.lock
```

Comment:
For the build-related targets, it certainly makes sense to place the lockfile into `$BUILD_DIR`. However, this leads to `$BUILD_DIR` getting created when doing  `make status`, which seems odd.

Reply:
That is true, but I don't see a reasonable alternative.
If I use a temporary file in `/tmp`, there's no good mechanism to get rid of it.

Reply:
One alternative would be to check in the conan.lock file into the root of the project. But I think that might just confuse people: when are we supposed to use that? Is it used by default?